packages/terra-popup/tests/jest/Popup.test.jsx

```diff
@@ -1,35 +1,37 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+/* eslint-disable import/no-extraneous-dependencies */
+import { shallowWithIntl, mountWithIntl } from 'terra-enzyme-intl';
```

Comment:
with latest changes we don't need intl wrapper modules for Jest as well

Reply:
Reverted as suggested.